camel-core/src/main/java/org/apache/camel/Exchange.java

```diff
@@ -170,6 +170,9 @@
     String LOOP_INDEX               = "CamelLoopIndex";
     String LOOP_SIZE                = "CamelLoopSize";
 
+    // Long running action (saga)
+    String SAGA_LONG_RUNNING_ACTION = "Long-Running-Action";
```

Comment:
Must it be `Long-Running-Action` ? Can it not be "CamelSagaLongRunningAction" like the others

Reply:
It could, but setting it to "Long-Running-Action" simplifies things a lot, since that is the same header used in the Microprofile implementation, so Camel routes would be able to interoperate naturally with them. So I prefer to stick with that if not an issue.

Reply:
Ah okay, maybe add a code comment about this so its easier to understand for us in the future